Makefile

```diff
@@ -124,6 +124,7 @@ install-themes:
 ## Compile Theme JSON Schema
 .PHONY: compile-themes-schema
 compile-themes-schema:
+	npm install -g @typespec/compiler
```

Comment:
Why do we need this?  Dependency installation should not be part of the compilation command

Reply:
I was thinking that users might not have compiler installed before running this script, but I think you are right.